org.kevoree.kcl/src/main/java/org/kevoree/kcl/impl/FlexyClassLoaderImpl.java

```diff
@@ -525,4 +525,18 @@ public FlexyClassLoaderImpl graphResourceOwnerResolution(KlassLoadRequest reques
         return result;
     }
 
+    @Override
+    public List<URL> getClassLoaderResources() {
+        return classpathResources.getResources();
+    }
+
+    @Override
+    public List<URL> getAllResources() {
+        List<URL> allResources = new ArrayList<URL>();
+        for (FlexyClassLoader classLoader : subClassLoaders) {
+            allResources.addAll(classLoader.getAllResources());
```

Comment:
Here we have a risk of loop. FlexyClassLoader can be organized has a graph, therefore resolution should be implemented using a loop-breaking mechanism such as the one used in Class resolution